src/Kovan-DssSpell.t.sol

```diff
@@ -1179,6 +1179,35 @@ contract DssSpellTest is DSTest, DSMath {
             calc_step:    0,
             calc_cut:     0
         });
+        afterSpell.collaterals["MATIC-A"] = CollateralValues({
+            aL_enabled:   true,
+            aL_line:      10 * MILLION,
+            aL_gap:       3 * MILLION,
+            aL_ttl:       8 hours,
+            line:         3 * MILLION, // TODO: is this correct? currently setting this according to dc-iam gap
+            dust:         10000, // TODO: seeing other tests set this to 100, maybe for supporitng tests
```

Comment:
Yeah, for testnet we'll want this at 100 so people can use it. It'll have to be updated when we port this to mainnet.

Reply:
done